src/components/mod.rs

```diff
@@ -16,15 +17,24 @@ pub struct Components {
 #[strum_discriminants(derive(EnumMessage, EnumIter))]
 /// What are you up to?
 pub enum ComponentsCommand {
-    #[strum_discriminants(strum(message = "download   -   Download components from account"))]
+    #[strum_discriminants(strum(
+        message = "download             -   Download components from account"
+    ))]
     /// Download components from account
     Download(self::download::AccountId),
     #[strum_discriminants(strum(
-        message = "deploy     -   Deploy components if code has changed"
+        message = "diff-code-deploy     -   Differences between component code for deployment"
+    ))]
+    /// Differences between component code for deployment
+    DiffCodeDeploy(self::diff_code_deploy::DiffCodeDeploy),
```

Comment:
Let's rename it to just `diff` / `Diff`, so it is `bos components diff` (taking inspiration from `git diff` command)

Reply:
Done